flow360/component/simulation/translator/solver_translator.py

```diff
@@ -709,6 +737,38 @@ def get_solver_json(
                 translated["turbulenceModelSolver"]["modelConstants"] = translated[
                     "turbulenceModelSolver"
                 ].pop("modelingConstants")
+
+            if not isinstance(model.transition_model_solver, NoneSolver):
+                # baseline dictionary dump for transition model object
+                translated["transitionModelSolver"] = dump_dict(model.transition_model_solver)
+                transition_dict = translated["transitionModelSolver"]
+                replace_dict_key(transition_dict, "typeName", "modelType")
```

Comment:
There are so many duplicated processing in NS and turb solver, in future we should abstract this out. It is fine not to change this in this PR. Just a note